app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt

```diff
@@ -132,14 +135,42 @@ class OngoingTopicListActivityTest {
     }
   }
 
+  @Test
+  fun testTopicPracticeFragment_loadFragment_changeOrientation_textIsCorrect() {
+    launch<OngoingTopicListActivity>(createOngoingTopicListActivityIntent(internalProfileId)).use {
+      onView(withId(R.id.ongoing_topic_list)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          0
+        )
+      )
+      onView(atPositionOnView(R.id.ongoing_topic_list, 0, R.id.topic_name_text_view)).check(
+        matches(
+          withText(containsString("Ratios and Proportional Reasoning"))
+        )
+      )
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.ongoing_topic_list)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          0
+        )
+      )̥
```

Comment:
If you notice this, this is not a bracket, this is some special character which is not in English. There must have been some typo while writing code. Please remove this.

Reply:
There is a `circle/dot` below bracket

Reply:
Removed.. Can you check now @rt4914